File: articles/transformational-leadership-organizational-change/c/transformational_leadership_score.c
/*
Transformational Leadership and Organizational Change
Simple C scoring demonstration.

Responsible-use scope:
Synthetic-data research, methods demonstration, institutional learning,
transformational-leadership review, change-system analysis, and reproducible
workflows only. Not for employee screening, hiring, promotion, compensation,
discipline, termination, workplace surveillance, individual performance
management, productivity ranking, leadership-loyalty scoring, dissent tracking,
or psychological assessment.
*/

#include <stdio.h>

double transformational_capacity_score(
    double vision_credibility,
    double motivational_alignment,
    double intellectual_stimulation,
    double developmental_support,
    double institutional_trust,
    double psychological_safety,
    double change_communication,
    double implementation_support,
    double meaning_making_quality,
    double resistance_intensity,
    double ambiguity_pressure,
    double change_fatigue
) {
    return
        0.12 * vision_credibility +
        0.11 * motivational_alignment +
        0.11 * intellectual_stimulation +
        0.10 * developmental_support +
        0.12 * institutional_trust +
        0.11 * psychological_safety +
        0.10 * change_communication +
        0.09 * implementation_support +
        0.10 * meaning_making_quality -
        0.08 * resistance_intensity -
        0.07 * ambiguity_pressure -
        0.08 * change_fatigue;
}

int main(void) {
    double score = transformational_capacity_score(84, 81, 80, 79, 82, 80, 83, 78, 82, 20, 22, 24);
    printf("Synthetic transformational capacity score: %.2f\n", score);
    printf("Responsible-use reminder: institutional learning only.\n");
    return 0;
}

File: articles/transformational-leadership-organizational-change/fortran/transformational_leadership_score.f90
! Transformational Leadership and Organizational Change
! Simple Fortran scoring demonstration.
!
! Responsible-use scope:
! Synthetic-data research, methods demonstration, institutional learning,
! and reproducible workflows only. Not for employment, productivity ranking,
! leadership-loyalty scoring, dissent tracking, or individual decisions.

program transformational_leadership_score_program
  implicit none
  real :: vision_credibility, motivational_alignment, intellectual_stimulation
  real :: developmental_support, institutional_trust, psychological_safety
  real :: change_communication, implementation_support, meaning_making_quality
  real :: resistance_intensity, ambiguity_pressure, change_fatigue
  real :: score

  vision_credibility = 84.0
  motivational_alignment = 81.0
  intellectual_stimulation = 80.0
  developmental_support = 79.0
  institutional_trust = 82.0
  psychological_safety = 80.0
  change_communication = 83.0
  implementation_support = 78.0
  meaning_making_quality = 82.0
  resistance_intensity = 20.0
  ambiguity_pressure = 22.0
  change_fatigue = 24.0

  score = 0.12 * vision_credibility + 0.11 * motivational_alignment + &
          0.11 * intellectual_stimulation + 0.10 * developmental_support + &
          0.12 * institutional_trust + 0.11 * psychological_safety + &
          0.10 * change_communication + 0.09 * implementation_support + &
          0.10 * meaning_making_quality - 0.08 * resistance_intensity - &
          0.07 * ambiguity_pressure - 0.08 * change_fatigue

  print *, "Synthetic transformational capacity score:", score
  print *, "Responsible-use reminder: institutional learning only."
end program transformational_leadership_score_program
